src/readers/morphologyASC.cpp

```diff
@@ -287,10 +287,19 @@ class NeurolucidaParser
                     lex_.consume();
                 }
                 lex_.consume(Token::RSPINE, "Must be end of spine");
+            } else if (skip_sexp(+id)) {
```

Comment:
why do you need this? I'm worried that this will skip something we shouldn't be skipping (ie: that doens't come right after a `LPAREN`.

Reply:
That's a good point. Thank you. I've rebuilt it.